src/routes/userdata.py

```diff
@@ -539,10 +541,14 @@ def start_login():
                 login_user(user)
                 return jsonify({"success": True})
             else:
+                print("test3")
                 return jsonify({"success": False})
-        except:
+        except Exception as E:
+            print(E)
+            print("test4")
             return jsonify({"success": False})
     else:
+        print("test5")
```

Comment:
Make sure once the full account set us is done to remove test print statements and replace with error messages